composer/datasets/imagenet.py

```diff
@@ -52,6 +55,11 @@ class ImagenetDatasetHparams(DatasetHparams, SyntheticHparamsMixin):
     resize_size: int = hp.optional("resize size. Set to -1 to not resize", default=-1)
     crop_size: int = hp.optional("crop size", default=224)
     use_ffcv: bool = hp.optional("whether to use ffcv for faster dataloading", default=False)
+    use_ffcv_randaugment: bool = hp.optional("Use randaugment with ffcv.", default=False)
+    randaugment_depth: int = hp.optional("randaugment depth.", default=2)
```

Comment:
I'm curious is it possible for the algorithms to patch the FFCV dataloader to insert these transformations? If we have to do this for every algorithm and dataset, I think it'll be very difficult to maintain.